packages/IMAPClient-Core.package/ICFileReadWriter.class/instance/write..st

```diff
@@ -0,0 +1,8 @@
+write file
+write: anICEmail
+	
+	(FileDirectory default / anICEmail maildirFileName) assureExistence.
```

Comment:
Can we have a test for that if its not existing? 🤔

Reply:
Writing tests is an issue which we know of, but we want to merge this first.